code_blocks/tools/customer-center-events-android-global.kt

```diff
@@ -0,0 +1,3 @@
+// Create and set the global listener
+val customerInfoListener = createCustomerCenterListener()
+sharedInstance.customerCenterListener = customerInfoListener 
```

Comment:
Shouldn't this say `Purchases`?
```suggestion
Purchases.sharedInstance.customerCenterListener = customerInfoListener 
```